includes/connect/class-wc-stripe-connect.php

```diff
@@ -267,6 +267,10 @@ private function save_stripe_keys( $result, $type = 'connect', $mode = 'live' )
 			$options[ $prefix . 'secret_key' ]          = $secret_key;
 			$options[ $prefix . 'connection_type' ]     = $type;
 			$options['pmc_enabled']                     = 'connect' === $type ? 'yes' : 'no'; // When not connected via Connect OAuth, the PMC is disabled.
```

Comment:
Thought for a follow-up/separate PR: maybe we should default to `''` or not even set `pmc_enabled` when we don't have ` connect` type.
```suggestion
			$options['pmc_enabled']                     = 'connect' === $type ? 'yes' : ''; // When not connected via Connect OAuth, defer the decision until we try to get a valid PMC.
```